nessie/src/test/java/org/apache/iceberg/nessie/TestBranchVisibility.java

```diff
@@ -101,14 +101,27 @@ public void testCatalogOnReference() {
     updateSchema(testCatalog, tableIdentifier2);
 
     // catalog created with ref points to same catalog as above
-    NessieCatalog refCatalog = initCatalog("test");
+    NessieCatalog refCatalog = initCatalog("test", null);
     testCatalogEquality(refCatalog, testCatalog, true, true, () -> { });
 
     // catalog created with hash points to same catalog as above
-    NessieCatalog refHashCatalog = initCatalog("main");
+    NessieCatalog refHashCatalog = initCatalog("main", null);
     testCatalogEquality(refHashCatalog, catalog, true, true, () -> { });
   }
 
+  @Test
+  public void testCatalogInitialHash() {
```

Comment:
End to end testcase with the USE reference command will be added in Nessie Integration test.